docs/reference/ingest/apis/enrich/put-enrich-policy.asciidoc

```diff
@@ -65,26 +65,10 @@ If you use {es} {security-features}, you must have:
 ==== {api-description-title}
 
 Use the put enrich policy API
-to create a new enrich policy.
-
-// tag::enrich-policy-def[]
-An *enrich policy* is a set of rules the enrich processor uses
-to append the appropriate data to incoming documents.
-An enrich policy contains:
-
-* The *policy type*,
-  which determines how the processor enriches incoming documents
-* A list of source indices
-* The *match field* used to match incoming documents
-* *Enrich fields* appended to incoming documents
-  from matching documents
-// end::enrich-policy-def[]
-
-
-===== Update an enrich policy
+to create a new <<enrich-policy,enrich policy>>.
 
 // tag::update-enrich-policy[]
-You cannot update an existing enrich policy.
+Once created, you can't update or change an enrich policy.
```

Comment:
This feels very delicate and a potential area that people accidentally override or change policies unexpectedly. should this be highlighted more in the form of a "WARNING" block?

Reply:
Good call. I agree. Added a warning admonition with 00a513c0327.